components/Footer.js

```diff
@@ -2,7 +2,7 @@ export default function Footer() {
 	const year = new Date().getFullYear();
 	return (
 		<footer>
-			<p>&copy; Erick Askim {year}</p>
+			<p className='footer'>&copy; Erick Askim {year}</p>
```

Comment:
I would suggest using BEM naming standards since you're using global styles.

Reply:
fixed